.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to Terraform Plugin SDK
```

Comment:
I think `CONTRIBUTING.md` is a bit premature at this point and should be removed altogether, but if you keep it, the content should be updated so it relevant to terraform-plugin-framework and not just terraform-plugin-sdk.

Reply:
Argh! Dunno how I missed this when modifying the SDKv2 repo scaffolding over. Sorry about that.